lib/br_danfe/danfe_lib/nfe_lib/infadic.rb

```diff
@@ -89,6 +90,29 @@ def fisco?
           @xml['infAdic/infAdFisco'].to_s.present?
         end
 
+        def format_dup_date(_det, dup_date)
+          dtduplicata = dup_date
+          "#{dtduplicata[8, 2]}/#{dtduplicata[5, 2]}/#{dtduplicata[0, 4]}"
+        end
+
+        def dup_content
+          dup = 0
+          value_dups = []
+
+          @xml.collect('xmlns', 'dup') do |det|
+            dup += 1
+            value = "#{det.css('nDup').text} - #{format_dup_date(det, det.css('dVenc').text)} - R$ #{BrDanfe::Helper.numerify(det.css('vDup').text.to_f)}"
+
+            if dup == 13
```

Comment:
Use a constante definida

```suggestion
            if dup == BrDanfe::DanfeLib::NfeLib::Dup::DUP_MAX_QUANTITY
```